### PR TITLE
Display server title correctly in dark mode

### PR DESCRIPTION
**Pull request by ann0see**
**Short description of changes**

Fixes the color of the Server name in dark mode on Linux (Gnome, Qt6, Debian 12). This should also work on macOS. Edit: it does. 

Without fix:

grafik

With fix:
grafik

Recording and mute myself doesn't change and looks OK:

grafik

CHANGELOG: CONDENSE with https://github.com/jamulussoftware/jamulus/pull/2833

**Context: Fixes an issue?**

**Does this change need documentation? What needs to be documented and how?**

No.

**Status of this Pull Request**

Ready for review. 

**What is missing until this pull request can be merged?**

## Checklist

-  [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
-  [x] I tested my code and it does what I want (See screenshots. I changed through the modes and tested recording. Not sure what else to test)
-  [x] My code follows the [style guide](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#source-code-consistency) 
-  [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
-  [x] I've filled all the content above

### CODE REVIEW

File: src/clientdlg.cpp [hunk 1]
```diff
@@ -1358,7 +1358,6 @@ void CClientDlg::SetGUIDesign ( const EGUIDesign eNewDesign )
             "                         image:          url(:/png/fader/res/ledbuttonpressed.png); }"
             "QCheckBox {              color:          rgb(220, 220, 220);"
             "                         font:           bold; }" );
-
```
Comment (pljones) [inline src/clientdlg.cpp:1361]: Unrelated delete of blank line?

**Review by pljones — APPROVED** [review]

File: src/clientdlg.cpp (hunk 1 above)
Comment (ann0see) [reply src/clientdlg.cpp:1361]: Yes.

**Comment by ann0see** [comment]
@gilgongo as it’s UI related, can you confirm that it’s ok? Darkmode in general seems to work ok. There might be some contrast issues I didn’t find yet. The server GUI might need some minor contrast adjustments for the Registered/Not registered messages.

**Comment by ann0see** [comment]
@gilgongo could you please review this PR?

**Comment by gilgongo** [comment]
Sorry - I wasn't sure if I was setting up dark mode correctly (I'm using Ubuntu 22.10) but it seemed OK to me.

**Review by gilgongo — APPROVED** [review]

**Comment by ann0see** [comment]
Ah ok. For me everything was fine under GNOME. So I believe it's ok. But I'm using Debian 12.

**Pull request merged**